api/models.py

```diff
@@ -1,3 +1,10 @@
 from django.db import models
 
+
 # Create your models here.
+
+
+class Bracket(models.Model):
+    title = models.CharField(max_length=100)
```

Comment:
Can we make the CharField 255? That seems to be sort of a standard for charfields that I've seen in class.